functions-methods.js

```diff
@@ -20,6 +36,27 @@
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
+function typeOfEmail( email ) {
+    //let validity = /\S+@\S+\.\S+/
+   // return validity.test( email )
+
+    if ( typeOfEmail.includes( '@novi' ))
+
+        console.log( 'Medewerker' )
+
+    if ( typeOfEmail.includes( '@novi-edication' ))
```

Comment:
Hier zou ik een `else if` gebruiken, want als je `@novi-education` in het e-mail adres hebt staan, dan zie je in je console dus 'Medewerker' en 'Student' staan